hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcConnectionRegistry.java

```diff
@@ -51,6 +51,9 @@
   /** Configuration key that controls the fan out of requests **/
   public static final String HEDGED_REQS_FANOUT_KEY = "hbase.client.bootstrap.hedged.fanout";
 
+  public static final String INITIAL_REFRESH_DELAY_SECS =
```

Comment:
nit: I think this could use a small comment (essentially what you added in the jira) to make it obvious.